plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/secrets/SecretsPluginConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.plugin.access.secrets;
+
+import static java.lang.String.join;
+
+public interface SecretsPluginConstants {
+    String REQUEST_PREFIX = "go.cd.secrets";
+    String _SECRETS_CONFIG_METADATA = "secrets-config";
+
+    String REQUEST_GET_PLUGIN_ICON = REQUEST_PREFIX + ".get-icon";
+    String REQUEST_GET_SECRETS_CONFIG_METADATA = join(".", REQUEST_PREFIX, _SECRETS_CONFIG_METADATA, ".get-metadata");
```

Comment:
`.get-metadata` should be `get-metadata` (without extra dot at the beginning), similarly `.get-view` and `.validate`.

Reply:
done